package.json

```diff
@@ -35,6 +35,7 @@
   "dependencies": {
     "axios": "^0.21.1",
     "babel-eslint": "^10.1.0",
+    "crypto": "^1.0.1",
```

Comment:
Is this https://www.npmjs.com/package/crypto ? That page says this library is deprecated and built into node now.